stripe/src/main/java/com/stripe/android/model/Card.java

```diff
@@ -308,9 +308,10 @@ public boolean validateCVC() {
         String cvcValue = cvc.trim();
 
         String updatedType = getType();
-        boolean validLength = ((updatedType == null && cvcValue.length() >= 3 && cvcValue.length() <= 4) ||
-                (AMERICAN_EXPRESS.equals(updatedType) && cvcValue.length() == 4) ||
-                (!AMERICAN_EXPRESS.equals(updatedType) && cvcValue.length() == 3));
+        boolean validLength =
+                ((updatedType == null && cvcValue.length() >= 3 && cvcValue.length() <= 4)
+                || (AMERICAN_EXPRESS.equals(updatedType) && cvcValue.length() == 4)
+                || (cvcValue.length() == 3));
```

Comment:
This is the real change -- all types, including AMEX are valid if length is three